rhea/build/boards/get_board.py

```diff
@@ -9,7 +9,7 @@
 from .xilinx import (Xula, Xula2, Xula2StickItMB, 
                      Pone, Anvyl, Mojo, Nexys, Atlys,
                      Zybo, Parallella, SX1, UFO400, XUPV2P)
-from .altera import (DE0Nano, DE0NanoSOC, DE0CV)
+from .altera import (DE0Nano, DE0NanoSOC, DE0CV, DE1SoC)
```

Comment:
Probably should change the board definition to `DE1SOC` to be consistent with the other terasic board definition names.